### ISSUE TITLE
Deleting a playlist have a too much "threatening" alert message

### ISSUE THREAD

**Issue by Nicryc** [jellyfin/jellyfin-web#3837]
**Describe The Bug**
When deleting a playlist a dialogue with the following message is displayed:

> **Delete Item?**
> Deleting this item will delete it from both the file system and your media library. Are you sure you wish to continue?

It makes think that media inside the playlist are going to be deleted too.

**Steps To Reproduce**

1. Create a playlist
2. Try to delete it

**Expected Behaviour**

The message should be more specific to playlists or less "threatening".

**System**
 - Platform: Windows 10
 - Browser: Firefox 103
 - Jellyfin Version: 10.8.1

**Comment by SegiH** [comment]
How about something like this ? "Deleting this playlist will delete the playlist file from both the file system and your media library. Are you sure you wish to continue?"

**Comment by Nicryc** [comment]
Is a playlist really a file ? Besides I don't think the playlist actually exists on the file system, it's clearly a Jellyfin abstraction. So I'd get rid of the file system part.

I would see a much simpler warning message: "Are you sure you wish to delete this playlist?". Maybe with the addition of a comforting message like: "Your media will stay untouched".

**Comment by SegiH** [comment]
Yes it is a file. A playlist created normally through Jellyfin saves an XML file or you can use a m3u8 playlist format. Either way, the playlist does exist on the file system so that's why I changed the wording to reflect that

**Comment by Nicryc** [comment]
Yeah obviously it exists somewhere but I mean it's not something the user will modify directly on the file system, copy, move or rename like he will do with a video or music file.

**Comment by SegiH** [comment]
I created the change request. The wording is now "Deleting this item will not affect you media. Are you sure you wish to delete this playlist?"

[PR #4038](https://github.com/jellyfin/jellyfin-web/pull/4038)

**Comment by thornbill** [comment]
If the playlist is an m3u8 file inside your media library does it get deleted?

**Comment by Nicryc** [comment]
Is it even possible to add a playlist by putting a m3u8 file in a media folder?

**Comment by SegiH** [comment]
Yes. I do exactly this. In fact, I wrote a simple app that creates an empty m3u8 file in a Playlist folder which is in my media library because Jellyfin creates an xml playlist by default and I prefer m3u8

**Comment by Nicryc** [comment]
Where do you place it? And how do you name it? Because I tested quickly I couldn't make it work.

**Comment by SegiH** [comment]
I created a folder called Playlists in music library and put my m3u8 playlists in this location. JF automatically picks it up

**Comment by Nicryc** [comment]
In Jellyfin, does it appears in the Music folder or in the Playlists folder?

**Comment by SegiH** [comment]
Playlists

Do you want to see a sample of one of my m3u8 files?

**Comment by Nicryc** [comment]
Yes that would be nice.

I tried to create a music library (because it seems to work only with music library which is why I didn't know it was possible since I don't have a music library) and my playlist appears in the Playlists folder but is empty. I've created my playlist using VLC.

**Comment by SegiH** [comment]
Here's the content from one of my m3u8 playlist files

#EXTM3U
#EXTALB:Immortal Beloved
#EXTART:Beethoven, Ludwig van
#EXTINF:180,Fur Elise
../Beethoven/Immortal Beloved/02 Fur Elise.mp3
#EXTART:Beethoven, Ludwig van
#EXTINF:900,Moonlight Sonata
../Beethoven/Misc/Moonlight Sonata.mp3
#EXTART:Carly Commando
#EXTINF:345,Everyday
../Carly Commando/00 Everyday.mp3
#EXTART:Schubert
#EXTINF:113,Moment Musicaux No. 3 in F Minor
../Schubert/Moment Musicaux No. 3 in F Minor.mp3
#EXTART:Tchaikovsky
#EXTINF:450,Waltz of the Flowers
../Tchaikovsky/Waltz of the Flowers.mp3
#EXTART:Vivaldi
#EXTINF:2519,Four Seasons
../Vivaldi/Four Seasons.mp3
#EXTALB:Greatest Hits
#EXTART:Mozart, Wolfgang Amadeus
#EXTINF:212,Rondo Alla Turca (Turkish March)
../Mozart/Greatest Hits/08 Rondo Alla Turca (Turkish Marc.mp3

**Comment by Nicryc** [comment]
Thanks, apparently it doesn't work with absolute file path (at least on Windows).

> If the playlist is an m3u8 file inside your media library does it get deleted?

So I just tested that. I put a m3u8 playlist file on my computer in `C:\Users\john\Music`, created a music library on Jellyfin with the previous path. The playlist appears in the Playlists folder of Jellyfin but **it's impossible to delete it** from Jellyfin, it's throwing an error saying Jellyfin doesn't have write permission on the folder.

**Comment by SegiH** [comment]
I ran into that same issue trying to delete a m3u8 playlist in Jellyfin.

**Comment by ciacca123** [comment]
Is this enhancement implemented? What does it take? Can someone help?

**Comment by SegiH** [comment]
It doesn't appear that this change has been implemented. The devs were working on a large scale endeavor and were not focused on resolving issues. I believe that they have completed this which will hopefully make updates easier and quicker
